overlord/devicestate/firstboot.go

```diff
@@ -70,6 +70,13 @@ func installSeedSnap(st *state.State, sn *snap.SeedSnap, flags snapstate.Flags)
 	return snapstate.InstallPath(st, &sideInfo, path, sn.Channel, flags)
 }
 
+func trivialSeeding(st *state.State, markSeeded *state.Task) []*state.TaskSet {
+	// give a chance to internal core config to run
```

Comment:
sounds a bit unenglish (maybe just me), maybe: `give the internal core config a chance to run` ?

Reply:
changed